bin/terminate-clients.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -ex
+
+psql -d postgres -U vimc <<EOF
+SELECT pg_terminate_backend(pg_stat_activity.pid)
+FROM pg_stat_activity
+WHERE pg_stat_activity.datname = 'montagu'
+AND pid <> pg_backend_pid();
+EOF
```

Comment:
I'm guessing this chunk of SQL is copied from an example somewhere. Could you maybe add a comment giving the URL? Or, if on the other hand this is all your own work, a comment or two to explain how it works would be nice. In particular, I'd like to know what `pid <> pg_backend_pid()` is doing. I'm guessing it excludes the process that is running this script, but I'm not sure how.

Reply:
yeah, this is totally stackoverflow driven development - I'll add a comment